tests/203_dbaas_policy_add_connection_namespace.robot

```diff
@@ -0,0 +1,68 @@
+*** Settings ***
+Documentation       To verify DBaaSPolicy
+Metadata            Version    0.0.1
+
+Library             SeleniumLibrary
+Resource            ../resources/keywords/dbaas_policy.resource
+
+Suite Setup         Configure DBaaS Policy Namespaces
+
+Suite Teardown      Tear Down The Test Suite
+Test Teardown       Tear Down DBaaSPolicy Test Case
+
+Force Tags          UI    DBaaS
+
+
+*** Test Cases ***
+Scenario: DBaaSInventories On Source Namespace Is Not Available Without ConnectionNamespace And With Role Permission
+    [Tags]    dbaas-test-34    smoke
+    Given User Creates DBaaSPolicy As OCP_LDAP_PROJECT_ADM Without ConnectionNamespace And Imports MongoDB Provider Account
+    When User Creates ${connectionNs} And Aligns admin Role To The OCP_DBAAS_USER
+    And User Aligns admin Role To ${DBaaSTestNS} For OCP_DBAAS_USER
+    Then DBaaSInventory Is Not Available For ${connectionNs}
+
+Scenario: DBaaSInventories On Source Namespace Is Not Available To ConnectionNamespace And Without RBAC Permission
```

Comment:
I think we can drop the 'And' here. It is a bit confusing...WDYT?

Reply:
IMHO, this should be fine as we already have more than the limit.